src/modules/content-files/content-files.c

```diff
@@ -277,8 +277,10 @@ static struct content_files *content_files_create (flux_t *h)
      */
     if (!(dbdir = flux_attr_get (h, "statedir")))
         dbdir = flux_attr_get (h, "rundir");
-    if (!dbdir)
+    if (!dbdir) {
         flux_log_error (h, "neither statedir nor rundir are set");
+        goto error;
+    }
```

Comment:
Thanks for catching this!  Wow don't know how I managed to do that not once but twice.